boards/shields/st_stm32f4dis_cam/boards/stm32l4r9i_disco.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&dma1 {
+	status = "okay";
+};
+
+&dmamux1 {
+	status = "okay";
+};
+
+#define MCO1_SEL_SYSCLK 1
+
+#define MCO1_PRE_DIV_4 2
```

Comment:
Not blocking: We should really get rid of these and provide definitions in `include/zephyr/dt-bindings/clock/stm32l4_clock.h` as done on other series.